website/docs/r/identitystore_group.html.markdown

```diff
@@ -0,0 +1,62 @@
+---
+subcategory: "IdentityStore"
```

Comment:
This might have to be `SSO Identity Store` in line with the data sources already there.

Reply:
Fixed